apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,123 @@
+import { Client } from "auth0";
+
+const USER_ID: string = "auth0|667c3d0c90b963e3671f411e";
+
+describe("Client API", () => {
+  const PORT: number = (Cypress.env("PORT") as number) || 8089;
+  let accessToken: string;
+  let m2mToken: string;
+  let newClientId: string;
+
+  const getTokens = () => {
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt-m2m")
+      .then((token: string) => {
+        if (token) {
+          m2mToken = token;
+        }
+      });
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt-client")
+      .then((token: string) => {
+        if (token) {
+          accessToken = token;
+        }
+      });
+  };
+
+  before(() => {
+    getTokens();
+    if (!accessToken) {
+      cy.loginM2MAuth0();
+      cy.wait(1500);
```

Comment:
Why do we need to wait?

Reply:
I don't remember why I added those. They don't seem to be necessary. I'll remove them.